src/app/services/contacts/contacts.ts

```diff
@@ -53,27 +53,33 @@ export class ContactsProvider {
 	}
 
 	public editContact(address: string, name: string): Observable<any> {
-		const contact = this.getContactByAddress(address);
-		if (!contact) {
+		if (!name) {
 			return throwError({
-				key: "CONTACTS_PAGE.CONTACT_NOT_EXISTS_ADDRESS",
-				parameters: { address },
+				key: "CONTACTS_PAGE.CONTACT_NAME_EMPTY",
 			} as TranslatableObject);
 		}
 
-		if (!name) {
+		if (!address) {
 			return throwError({
-				key: "CONTACTS_PAGE.CONTACT_NAME_EMPTY",
+				key: "CONTACTS_PAGE.CONTACT_ADDRESS_EMPTY",
```

Comment:
This key needs to be added to the locale file (src/assets/i18n/en.json).

Reply:
Done 👌